src/quo2/components/wallet/account_overview/style.cljs

```diff
@@ -0,0 +1,56 @@
+(ns quo2.components.wallet.account-overview.style
+  (:require [quo2.foundations.colors :as colors]))
+
+(def account-overview-wrapper
+  {:width           335
+   :height          78
+   :align-items     :center
+   :justify-content :center})
+
+(def row {:flex-direction :row})
+
+(def account-name
+  {:margin-bottom 3
+   :color         (colors/theme-colors colors/neutral-100 colors/white)})
```

Comment:
same here? I think these are the default colors 🤔

Reply:
Cleaned